src/commands/init.ts

```diff
@@ -32,7 +41,7 @@ export abstract class InitCommand extends Command {
       description: 'Path where the component file should be written to',
       default: 'architect.yml',
     }),
-    account: flags.string({
+    account: flags.string({ // TODO: should this be required? why can't we use this offline or without an account?
```

Comment:
Should we make this available offline by making an account optional?

Reply:
fine by me